gradle.properties

```
@@ -4,4 +4,4 @@ org.gradle.parallel = true
 kotlin.incremental = true
 kotlin.code.style = official
 
-version = 0.1.3
+version = 0.1.4
```

This can stay as 0.1.3, as it isn't uploaded anywhere
```suggestion
version = 0.1.3
```